test/unit/util/zip.js

```diff
@@ -208,3 +208,33 @@ describe('zipPart streamer', () => {
   });
 });
 
+describe('zipStreamToFiles()', () => {
+  it('should not conflate metadata & file data', (done) => {
+    const part = zipPart();
+
+    zipStreamToFiles(zipStreamFromParts(part), (err, result) => {
+      // eslint-disable-next-line keyword-spacing
+      if(err) return done(err);
```

Comment:
Would be nice to make it `if (err)` with a space instead of disabling the ESLint rule. We could probably set up ESLint to auto-fix things like this if that'd be helpful.

Reply:
:+1: this was copied from elsewhere in this file.  I've opened a PR to clean up similar eslint-disable directives: https://github.com/getodk/central-backend/pull/1328

Reply:
Thank you!